test/ui/LoginTest.js

```diff
@@ -0,0 +1,122 @@
+var webdriver = require('selenium-webdriver');
+var chai = require("chai");
+var expect = chai.expect;
+
+describe('Login Page', function() {
+    this.timeout(50000); // Set timeout to 5 seconds
```

Comment:
You have set timeout to 50seconds -> Should this be 5 seconds as per the comment? Furthermore, I believe this file has the wrong extension. Should it not be ".ts". Same for LoginTest.js